ocs_ci/deployment/deployment.py

```diff
@@ -508,6 +508,7 @@ def deploy_ocs_via_operator(self):
                 ] = self.DEFAULT_STORAGECLASS
 
             ocs_version = float(config.ENV_DATA['ocs_version'])
+            ocp_version = get_ocp_version()
```

Comment:
I know we discuss it some time back that string comparation also works, but can we use float here as well?
```suggestion
            ocp_version = float(get_ocp_version())
```

Reply:
Done